src/uct/base/uct_iface.c

```diff
@@ -290,6 +310,10 @@ int uct_iface_is_reachable_v2(const uct_iface_h tl_iface,
 {
     const uct_base_iface_t *iface = ucs_derived_of(tl_iface, uct_base_iface_t);
 
+    if (params->info_string != NULL) {
```

Comment:
use UCS_PARAM_VALUE and do it only if info_str_len>0

Reply:
without checking that it's not NULL?

Reply:
in addition to checking for NULL